src/MeshlessFV/MfvMusclSimulation.cpp

```diff
@@ -148,12 +143,13 @@ void MfvMusclSimulation<ndim>::MainLoop(void)
 
 
 
-  // Calculate terms due to self-gravity
-  if (mfv->self_gravity == 1) {
+  // Calculate terms due to self-gravity / stars
+  if (mfv->self_gravity == 1 || nbody->Nnbody > 0) {
```

Comment:
Yes, we could do this.  I guess the problem is the SPH forces are all computed in the same routine whereas the meshless is effectively split between hydro and gravity.  So, perhaps we should write a 'ComputeStarGravForces' routine common to all hydro routines rather than bury the in the NeigbhourSearch routines that we currently do?